fa_linux_pc:
  transform_prefix: careculino_
  bb_map_odom_base_link_transforms:
    publish_transforms_from_start: true
    publish_map_odom_base_link_without_prefix: false
    publish_world_tf: true
    publish_world_tf_without_prefix: false
    world_transform:
      frame_id: world
      child_frame_id: map
      translation:
        x: 0.0
        y: 0.0
        z: 0.0
      rotation:
        r: 0.0
        p: 0.0
        y: 0.0
    map_odom_transform:
      frame_id: map
      child_frame_id: odom
      translation:
        x: 0.0
        y: 0.0
        z: 0.0
      rotation:
        r: 0.0
        p: 0.0
        y: 0.0
    odom_base_link_transform:
      frame_id: odom
      child_frame_id: base_link
      translation:
        x: 0.0
        y: 0.0
        z: 0.0
      rotation:
        r: 0.0
        p: 0.0
        y: 0.0
